database-plugins/widgets/Database-connector.json

```diff
@@ -7,6 +7,14 @@
     {
       "label": "Basic",
       "properties": [
+        {
+          "widget-type": "plugin-list",
+          "label": "Plugin Name",
```

Comment:
can we call this JDBC Driver name? It is unclear what `Plugin Name` means. This is a problem with the current Database plugins as well.